racetime/static/racetime/script/race.js

```diff
@@ -8,6 +8,32 @@ function Race() {
     this.messageIDs = [];
     this.notify = false;
 
+    try {
+        var userSelection = document.getElementsByClassName("scrollwarning");
+        for (let i = 0; i < userSelection.length; i++) {
+            userSelection[i].addEventListener("click", function() {
+                var $messages = $('.race-chat .messages');
+                $('.race-chat').removeClass('scrollwarning');
```

Comment:
Bit confused by the mixed use of vanilla JS and jQuery here. Ideally I think we should just use jQuery for these bits, but it's not essential. 

Also please try and stick to using 'single quotes' instead of "double quotes", for consistency.

Reply:
For the single quotes vs double quotes I just tended to roll double quotes because PEP 257 called for it. I blame that my work is converting to double quotes right now.

Reply:
For the single quotes vs double quotes I just tended to roll double quotes because PEP 257 called for it. I blame that my work is converting to double quotes right now.
For the JS vs jQuery, does it show I'm not primarily a webdev :P